src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java

```diff
@@ -113,4 +114,67 @@ public void onReceiveSlashKeeperAddShouldReturnErrorMessageIfOccurException() th
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.text").value("something went wrong"));
     }
+
+    @Test
+    public void onReceiveSlashCommandGetKeeperDirectionsReturnEmptyRichMessage() throws Exception {
```

Comment:
Dou you know about unit test naming convention. Ivan wrote in slack chat 2-3 weeks ago.